content/applications/websites/livechat/chatbots.rst

```diff
@@ -137,42 +138,48 @@ help qualify conversations before they reach live human operators.
 Free Input/Multi-Line
 *********************
 
-The free input step allows visitors to respond to questions without providing pre-written responses.
-Information provided in these responses is stored in the chat transcripts.
+The *free input* step allows visitors to respond to questions without providing pre-written
+responses. Information provided in these responses is stored in the chat transcripts.
 
 Choose between :guilabel:`Free Input` and :guilabel:`Free Input (Multi-Line)` depending on the type
-and amount of information being requested from the visitor.
+and amount of information the visitor is asked to provide.
 
 Create Lead
 ***********
 
 This step creates a lead in the *CRM* application. Select an option from the :guilabel:`Sales Team`
```

Comment:
Totally optional, but you could use the abbreviation markup to spell out what CRM means/stands for.